aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -67,6 +66,22 @@
   - rm -rf /home/ubuntu/git/beam/test/input/sf-light/r5/network.dat
   - ln -sf /var/log/cloud-init-output.log ./cloud-init-output.log
   - hello_msg=$(printf "Run Started \\n Run Name** $TITLED** \\n Instance ID %s \\n Instance type **%s** \\n Host name **%s** \\n Web browser ** http://%s:8000 ** \\n Region $REGION \\n Batch $UID \\n Branch **$BRANCH** \\n Commit $COMMIT" $(ec2metadata --instance-id) $(ec2metadata --instance-type) $(ec2metadata --public-hostname) $(ec2metadata --public-hostname))
+   - start_json=$(printf "{
+      \\"command\\":\\"add\\",
```

Comment:
Should this be create?

Reply:
It means add row to the corresponding sheet, we got beam and pilates